site/public/apps/rheum-derm-immune-atlas/index.html

```
@@ -0,0 +1,646 @@
+        <div class="metric"><strong id="nCanonical">–</strong><span>canonical network programs</span></div>
+      </div><div class="cutoff"><b>Evidence snapshot:</b> <span id="cutoff"></span><br>Designed for education and hypothesis organization—not prescribing, diagnosis or patient-specific decision support.</div></aside>
+    </div>
+
+    <div class="grid two">
+      <section class="panel"><div class="panel-head"><div><h3>Immune-pathway intensity map</h3><div class="panel-sub">Maximum mechanistic significance (1–5) by condition and canonical pathway family.</div></div><div class="controls"><select id="heatmapSort" class="control"><option value="default">Condition order</option><option value="alphabetic">Alphabetic</option><option value="inflammation">Inflammation intensity</option></select></div></div><div class="chart" style="height:570px"><canvas id="heatmap" aria-label="Pathway heatmap"></canvas></div><div id="heatmapLegend" class="legend"></div></section>
+      <div class="grid">
+        <section class="panel"><div class="panel-head"><div><h3>Selected condition pathway fingerprint</h3><div class="panel-sub">Relative importance of the strongest canonical axes.</div></div><select id="radarCondition" class="control"></select></div><div id="radar" class="svg-chart"></div></section>
+        <section class="panel"><div class="panel-head"><div><h3>Evidence distribution</h3><div class="panel-sub">Grades across pathway, mechanism and benefit rows.</div></div></div><div class="evidence-box"><div class="donut-wrap"><svg id="evidenceDonut" viewBox="0 0 120 120"></svg><div class="donut-center"><div><strong id="evidenceN">0</strong><span>graded rows</span></div></div></div><div id="evidenceLegend"></div></div></section>
+      </div>
+    </div>
+    <section class="panel" style="margin-top:16px"><div class="panel-head"><div><h3>Condition library</h3><div class="panel-sub">Open a module for physiology, causal pathophysiology, manifestations, biomarkers and treatment map.</div></div><input id="conditionCardSearch" class="control" type="search" placeholder="Filter conditions"></div><div id="conditionCards" class="condition-grid"></div></section>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)